src/UKHO.S100PermitService.Common/Providers/CacheProvider.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.Extensions.Caching.Memory;
+using System.Diagnostics.CodeAnalysis;
+
+namespace UKHO.S100PermitService.Common.Providers
+{
+    [ExcludeFromCodeCoverage]
+    public class CacheProvider : ICacheProvider
```

Comment:
Can we write Unit Test for it?

Reply:
Extension methods can not be intercepted since they're static. Hence excluded from code coverage